### DISCUSSION TITLE
How to configure custom ca certificates or disable ssl ?

### DISCUSSION THREAD

**Discussion by SurjitSahoo** [coreybutler/nvm-windows#1062] [Support]
In my organisation, ca certificates and network traffic is controlled with zscaler tool, so any tool that doesn't directly use OS CA certificates such as `npm` get TLS handshake error. With npm we can configure it to use the ZScaler provided certificates to resolve the issue. However I'm not able to find any solution for `nvm`

I get the following error. How to resolve this?
```
> nvm install 20

Could not retrieve https://nodejs.org/dist/latest-v20.x/SHASUMS256.txt.


Get "https://nodejs.org/dist/latest-v20.x/SHASUMS256.txt": net/http: TLS handshake timeout
```

**Comment by coreybutler** [comment]
That's due to a network error. The Node website was undergoing updates. Try again.

Related: #1064.

**Reply by coreybutler** [reply]
Side note - this happens relatively often due to how Node releases the software. We are building a more robust service for [Runtime](https://github.com/coreybutler/nvm-windows/wiki/Runtime) that will not experience the same file/resource "thrashing" that occurs during the Node.js release process.

**Comment by richenyadav001** [comment]
In Linux based os:
As nvm is using curl for all type of network activities. And from nvm there is no option to ignore strict ssl verification (till now). 

To set up nvm with cert you need to setup curl with certificate and for curl there is one environment variable which was checked by curl whenever it has to do some network activity.

On windows you need to set this env
CURL_CA_BUNDLE="/path_of_cert/org.cer"

There could something similar to this on windows to
